### PR TITLE
chore(curriculum): no quotes in code strings

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

As part of our i18n standards, we don't wrap strings in quotes when we use code tags.

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/record-collection.md [hunk 1]
```diff
@@ -21,8 +21,8 @@ Complete the function using the rules below to modify the object passed to the f
 
 -   Your function must always return the entire `records` object.
 -   If `value` is an empty string, delete the given `prop` property from the album.
--   If `prop` isn’t `"tracks"` and `value` isn't an empty string, assign the `value` to that album’s `prop`.
--   If `prop` is `"tracks"` and value isn’t an empty string, add the `value` to the end of the album’s `"tracks"` array. You need to create this array first if the album does not have a `"tracks"` property.
+-   If `prop` isn’t `tracks` and `value` isn't an empty string, assign the `value` to that album’s `prop`.
+-   If `prop` is `tracks` and value isn’t an empty string, add the `value` to the end of the album’s `tracks` array. You need to create this array first if the album does not have a `tracks` property.
```
Comment (ShaunSHamilton) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/record-collection.md:25]:
```suggestion
-   If `prop` isn't `tracks` and `value` isn't an empty string, assign the `value` to that album's `prop`.
-   If `prop` is `tracks` and value isn't an empty string, add the `value` to the end of the album's `tracks` array. You need to create this array first if the album does not have a `tracks` property.
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
On the subject of quotation marks...

---

Aside: Personally, I prefer a string being represented as `"string"`, as it helps differentiate between a code token (e.g. variable) named `string`. _not relevant for this pr_

**Comment by naomi-lgbt** [comment]
We explicitly decided against that when we were first establishing our i18n flows. Can I remember why? Nope. But it was a big thing.

**Review by ShaunSHamilton — APPROVED** [review]
WMSIANH 👍

**Review by gikf — APPROVED** [review]
LGTM :+1:

**Pull request merged**